pages/sdk/crons.mdx

```diff
@@ -69,20 +69,20 @@ Then `map` the pairs and `apply` each pair to `let`, which results in defining t
   Feature added in contract version 0.28.14
 </Callout>
 
-`if` can create a conditional statement. If the second item evaluates `true` it executes the third item.
+`if` can create a conditional statement. If the second item evaluates `true`, it executes the third item.
```

Comment:
`if` creates a conditional statement. If the second item evaluates `true`, it executes the third item.